portfolio/src/main/webapp/style.css

```diff
@@ -4,17 +4,22 @@ body {
     background-repeat: no-repeat;
     background-attachment: fixed;
     background-size: auto;
+    margin:0px;
+    padding:0px;
 
     
 }
 
-div:not(.header) {
+div:not(.header, ) {
```

Comment:
I don't think this added comma needs to be here? If it does though feel free to correct me! CSS is weird sometimes